# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bin86/x86_64.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Compile fixes x86-64 - imported from ROCK Linux.

  - Rene Rebe <rene@exactcode.de>

diff -urN bin86-0.16.14-orig/ld/x86_aout.h bin86-0.16.14/ld/x86_aout.h
--- bin86-0.16.14-orig/ld/x86_aout.h	2004-11-07 20:07:31.506442000 +0100
+++ bin86-0.16.14/ld/x86_aout.h	2004-11-07 20:07:41.106982496 +0100
@@ -12,6 +12,9 @@
 #if defined(i386) || defined(__BCC__) || defined(MSDOS)
 typedef long Long;
 #define __OUT_OK 1
+#elif defined(__x86_64__)
+typedef int Long;
+#define __OUT_OK 1
 #else
 typedef char Long[4];
 #endif
